### PR TITLE
Move folders created independent agents

### PR DESCRIPTION
**Pull request by mrodm**
Moved directories created by test executions using independent agents under profile directory:
- service logs: `~/.elastic-package/profiles/default/service_logs/agent-nginx-access`
- deployer files: `~/.elastic-package/profiles/default/deployer/docker-agent-nginx-access`

Example with test package `nginx`:
```
 $ ls -l  ~/.elastic-package/profiles/default/deployer/ ~/.elastic-package/profiles/default/service_logs/
/home/user/.elastic-package/profiles/default/deployer/:
total 4
drwxr-xr-x 2 user user 4096 abr 18 16:18 docker-agent-nginx-access

/home/user/.elastic-package/profiles/default/service_logs/:
total 4
drwxr-xr-x 2 user user 4096 abr 18 16:18 agent-nginx-access

```

### CODE REVIEW

File: internal/configuration/locations/locations.go [hunk 1]
```diff
@@ -73,21 +72,11 @@ func (loc LocationManager) DeployerDir() string {
 	return filepath.Join(loc.stackPath, deployerDir)
 }
 
-// StackDir returns the stack directory location
-func (loc LocationManager) StackDir() string {
-	return filepath.Join(loc.stackPath, stackDir)
-}
-
```
Comment (mrodm) [inline internal/configuration/locations/locations.go:80]: Not used anywhere.

File: internal/configuration/locations/locations.go [hunk 2]
```diff
@@ -73,21 +72,11 @@ func (loc LocationManager) DeployerDir() string {
...
 // PackagesDir returns the packages directory location
 func (loc LocationManager) PackagesDir() string {
 	return filepath.Join(loc.stackPath, stackDir, packagesDir)
 }
 
-// KubernetesDeployerDir returns the Kubernetes Deployer directory location
-func (loc LocationManager) KubernetesDeployerDir() string {
-	return filepath.Join(loc.stackPath, kubernetesDeployerDir)
-}
-
```
Comment (mrodm) [inline internal/configuration/locations/locations.go:90]: Not used anywhere.

**Review by jsoriano — APPROVED** [review]
LGTM, but I wonder if we could use randomized tmp directories for these directories.

**Comment by mrodm** [comment]
> LGTM, but I wonder if we could use randomized tmp directories for these directories.

I used the ID created for each test to add some randomness in the directory names (https://github.com/elastic/elastic-package/pull/1790/commits/36a2cfbbdcbcf201704a9eaaebd3698ed28afffb). I prefer to keep those folder names with some known names in advance (e.g. package, data stream, ...) in case the users need to check the contents of some of them.

**Review by jsoriano — APPROVED** [review]

**Pull request merged**